spaces/domains.json

```
@@ -162,4 +162,5 @@
   "vote.macaronswap.finance": "macaronswap.eth",
   "vote.resdx.io": "resdx.eth",
   "snapshot.solos.so": "solos.eth"
+  "vote.hpb.io": "hpbdao.eth"
```

A comma is missing at the end of the line above your sub-domain.
```suggestion
  "snapshot.solos.so": "solos.eth",
  "vote.hpb.io": "hpbdao.eth"
```